### PR TITLE
chore: apply ux feedback

### PR DESCRIPTION
**Pull request by JessicaSachs**
- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/frontend-shared/src/locales/en-US.json [hunk 1]
```diff
@@ -174,8 +174,8 @@
       "bodySuccess": "You are now logged in as {0}.",
       "bodyError": "An error occurred while attempting to log in:",
       "bodyBrowserError": "Cypress was unable to open your installed browser. To continue logging in, please paste this URL into your web browser:",
-      "dashboard": "Cypress Dashboard Service",
-      "titleInitial": "Log In To Cypress",
+      "cloud": "Cypress Cloud Service",
```
Comment (pstakoun) [inline packages/frontend-shared/src/locales/en-US.json:177]: Are we using "Cloud" in this release? I think this is still very much up in the air and may cause confusion if we haven't changed this verbiage throughout services before releasing this.

**Review by marktnoonan — APPROVED** [review]
Pulled this branch and the changes all seem to work just fine.

**Review by marktnoonan — CHANGES_REQUESTED** [review]
Actually noticed some inconsistent font weights in the header, not sure why we are bold for Docs when focused (and Upgrade always) but not for Browser and Login, but it looks off
Screen Shot 2021-12-07 at 2 46 00 PM
Screen Shot 2021-12-07 at 2 45 54 PM
Screen Shot 2021-12-07 at 2 45 49 PM

**Review by marktnoonan — DISMISSED** [review]

**Comment by marktnoonan** [comment]
I made minor changes to have consistent fonts in the header and resolve some conflicts where @ZachJW34 had made changes that overlapped.

**Pull request merged**